networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc

```diff
@@ -48,6 +48,14 @@ include::modules/virt-example-ethernet-nncp.adoc[leveloffset=+2]
 
 include::modules/virt-example-nmstate-multiple-interfaces.adoc[leveloffset=+2]
 
+include::modules/virt-example-host-vrf.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../networking/multiple_networks/about-virtual-routing-and-forwarding.adoc#cnf-about-virtual-routing-and-forwarding_about-virtual-routing-and-forwarding[About virtual routing and forwarding]
+* xref:../../networking/metallb/metallb-configure-bgp-peers.adoc#nw-metallb-bgp-peer-vrf_configure-metallb-bgp-peers[Configuring a BGP peer to advertise a service through a VRF]
```

Comment:
Looks like this links to a section called "Exposing a service through a network VRF". Can you update to use that title (there are a few other places that use this same xref as well)